### PR TITLE
adding myself as a supporter

### PR DESCRIPTION
**Pull request by turboMaCk**
:crossed_fingers:

### CODE REVIEW

**Comment by pesehr** [comment]
Please add your name based on the alphabet order

**Comment by turboMaCk** [comment]
I hope I get it right :eyes:

File: README.md [hunk 1]
```diff
@@ -503,6 +503,7 @@ You can:
 - Mahuton from Benin 🇧🇯 ([Mahuton](https://github.com/Mahuton))
 - Marco Aceti from Italy 🇮🇹 ([MarcoBuster](https://github.com/MarcoBuster))
 - Mariam Arab from Canada 🇨🇦 ([mariamarab](https://github.com/mariamarab))
+- Marek Fajkus from Czech Republic 🇨🇿 ([turboMack](https://github.com/turboMaCk))
```
Comment (turboMaCk) [inline README.md]: seems rebase added someone above me again? I hope it's not a big deal. Now I'm true off by one master....

Comment (pesehr) [reply README.md]: Don't worry!
We sort the list to avoid conflicts. 
The exact order doesn't matter

**Pull request merged**